ctpolicy/chromepolicy.go

```diff
@@ -15,10 +15,17 @@
 package ctpolicy
 
 import (
+	"time"
+
 	"github.com/google/certificate-transparency-go/loglist3"
 	"github.com/google/certificate-transparency-go/x509"
 )
 
+const (
+	minOperators = 2                   // minimum number of distinct CT log operators that issue an SCT.
+	dayInSeconds = 86400 * time.Second // number of seconds in a day.
```

Comment:
The `dayInSeconds` naming is a bit confusing. It is assigned with `time.Second` but the type is actually `time.Duration`.

Reply:
Changed the name to `dayDuration` to more accurately describe the `const`. I have also modified the comment accordingly.